ios/Surveys/Sources/Presentation/Modules/Home/HomeView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  HomeView.swift
+//  Surveys
+//
+//  Created by Su Ho on 03/11/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    var body: some View {
```

Comment:
same here.
```suggestion
struct HomeView: View {

    var body: some View {
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/97/commits/53bd127fdd575af6b3e1af2f8515a6b72a02f44b